src/test/java/codeu/model/store/basic/UserStoreTest.java

```diff
@@ -74,6 +75,16 @@ public void testAddUser() {
     Mockito.verify(mockPersistentStorageAgent).writeThrough(inputUser);
   }
 
+  public void testUpdateUserPhoto() throws PersistentDataStoreException {
```

Comment:
Not sure I understand this test completely. I also can't read the rest of line 79 for some reason. Can you see if you can move the URL to the next line?

Reply:
+    User inputUser = new User(UUID.randomUUID(), "test_username", "test_password", Instant.now(), "https://i1.wp.com/www.winhelponline.com/blog/wp-content/uploads/2017/12/user.png?resize=256%2C256&quality=100");
this is line 79! :)